cmd/cluster-init/generate_job.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+const (
+	LatestImage   = "registry.ci.openshift.org/ci/applyconfig:" + api.LatestReleaseName
```

Comment:
The `:latest` here is a tag, not a release name. This is technically an incorrect usage of the `api.LatestReleaseName` constant. I would suggest you to use `ServiceDomainAPPCIRegistry` for the registry, then append `ci/applyconfig:latest` to it.